app/models/federated_protocols/saml.rb

```diff
@@ -44,7 +44,7 @@ def service_provider
     end
 
     def biometric_comparison_required?
-      false
+      Vot::AcrComponentValues.satisfies?(ial, :biometric_comparison)
```

Comment:
(author's note): The change here is incorrect. This is solely to track this as a bit of code that should be removed or fixed to support the new biometric service.

Reply:
This API is removed by an upstream PR

Reply:
removed.